src/components/CourseInfo/CourseInfo.module.css

```diff
@@ -0,0 +1,34 @@
+.courseCard{
+    border: 1px solid #e0e0e0;
+    border-radius: 5px;
+    background: #F7F7F7;
+    margin-bottom: 20px;
+    padding: 20px;
+    box-shadow: 0px 2px 3px #aaa;
+
+}
+.courseCard .actions {
+    display: flex;
+    justify-content: flex-end;
+  }
+  
+  .courseCard h2 {
+    margin-top: 0;
+  }
+  
+  .courseCard p {
+    margin-bottom: 0;
+  }
+  
+  .courseCard .author {
+    font-size: 14px;
+    color: #888;
+    margin-bottom: 20px;
+  }
+
+  .courseCard > button {
```

Comment:
You can use nested rules in css modules:
```css
.courseCard{
    border: 1px solid #e0e0e0;
    border-radius: 5px;
    background: #F7F7F7;
    margin-bottom: 20px;
    padding: 20px;
    box-shadow: 0px 2px 3px #aaa;

    .actions {
      display: flex;
      justify-content: flex-end;  
    }

    h2 {
      margin-top: 0;
    }

    p {
      margin-bottom: 0;
    }

    > button {
      margin-top: 1rem;
    }

    &:hover {
      /* Some other styles */
    }
}

```

Reply:
I've used to nested styles. Thank you.